lab-shelly/test/greet-test.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const assert = require('assert');
+const greeting = require('../lib/greet.js');
+
+describe('greet.js', function() {
+  describe('#greet', function(){
+    it('should return hello with name', function(){
+      assert.equal(greeting.greet('shelly'), 'hello shelly' );
+    });
+    it('should return a string', function() {
+      assert.equal(typeof greeting.greet('shelly'), 'string');
+    });
```

Comment:
Also note that you can test your argv functionality as well. 
assert.equal(greeting.greet(process.argv[2]), 'hello process.argv[2]');